src/styles/polygons/polygons_vertex.glsl

```diff
@@ -77,27 +79,36 @@ void main() {
     vec4 position = vec4(a_position.xy, a_position.z / TANGRAM_HEIGHT_SCALE, 1.); // convert height back to meters
 
     #ifdef TANGRAM_EXTRUDE_LINES
+        // vec2 extrude = a_extrude.xy / 256.; // values have an 8-bit fraction
         vec2 extrude = a_extrude.xy / 256.; // values have an 8-bit fraction
         float width = a_extrude.z;
         float dwdz = a_extrude.w;
-
-        // Adjust line width based on zoom level, to prevent proxied lines from being either too small or too big.
-        // "Flattens" the zoom between 1-2 to peg it to 1 (keeps lines from prematurely shrinking), then interpolate
-        // and clamp to 4 (keeps lines from becoming too small when far away).
+        vec2 offset = a_offset.xy; // values have an 8-bit fraction;
```

Comment:
I think 8-bit fraction comment is no longer accurate since we're not multiplying/un-multiplying this value anymore